PYME/IO/tabular.py

```diff
@@ -535,7 +535,7 @@ def getInfo(self):
 class MatfileColumnSource(TabularBase):
     _name = "Matlab Column Source"
     
-    def __init__(self, filename):
+    def __init__(self, filename, columnnames=None):
```

Comment:
The column mapping stuff should be in a subclass of `MatfileColumnSource` or better yet, just load using the old/existing `MatfileColumnSource` and then use a `tabular.MappingFilter` to re-map column names. The rationale for the second approach is to separate VisGUI loading (which requires magic column names) from more general loading of tabular data which really doesn't care what the column names are.